BraveRewardsUI/Publisher/PublisherSummaryView.swift

```diff
@@ -68,6 +75,45 @@ class PublisherSummaryView: UIView {
       $0.height.equalTo(40.0)
     }
   }
+  
+  func addMonthlyTipView() {
+    if monthlyTipView == nil {
+      monthlyTipView = DetailActionableRow().then {
+        $0.textLabel.text = Strings.TipSiteMonthly
+        let batAttrString = NSAttributedString(
+          string: " BAT",
+          attributes: [
+            .foregroundColor: Colors.grey200,
+            .font: UIFont.systemFont(ofSize: 14.0)
+          ]
+        )
+        let batAmountAttrString = NSMutableAttributedString(
+          string: BATTipOption(value: 5).displayString,
+          attributes: [
+            .foregroundColor: Colors.purple300
+          ]
+        )
+        batAmountAttrString.append(batAttrString)
+        $0.detailButton.setAttributedTitle(batAmountAttrString, for: .normal)
+        $0.onTapCallback = {
+          // Show BAT amount selection screen.
+          
+          // FIXME:   Send correct BAT amounts ie either default or publisher listed.
+          let options = [
+            BATTipOption(value: 1),
+            BATTipOption(value: 5),
+            BATTipOption(value: 10)
+          ]
+          
+          let optionsVC = OptionsSelectionViewController(options: options, selectedOptionIndex: 1, optionSelected: { index in
```

Comment:
Shouldn't be creating this here anyways, its not the view's responsibility, its the controllers.